From b5aedd7844195a3ddae70f02de503636097a8ddb Mon Sep 17 00:00:00 2001
From: Seimizu Joukan <joukan.seimizu@jp.panasonic.com>
Date: Fri, 20 Aug 2021 15:56:37 +0900
Subject: [PATCH] Enable using python3

Python2 is not supported anymore since Yocto dunfell, 
but mesa-18.2.0 depends on python2.

Upstream-Status: Inappropriate [embedded specific]
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 42e1582..8c591ce 100644
--- a/configure.ac
+++ b/configure.ac
@@ -124,7 +124,7 @@ AC_PROG_GREP
 AC_PROG_NM
 AM_PROG_AS
 AX_CHECK_GNU_MAKE
-AC_CHECK_PROGS([PYTHON2], [python2.7 python2 python])
+AC_CHECK_PROGS([PYTHON2], [python3 python2 python])
 AC_PROG_SED
 AC_PROG_MKDIR_P
 
-- 
2.17.1

